modules/local/sentieon/bqsr.nf

```diff
@@ -0,0 +1,78 @@
+process SENTIEON_BQSR {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    input:
+    tuple val(meta), path(bam), path(bai), path(score), path(score_idx)
+    path fasta
+    path fai
+    path known_dbsnp
+
+    output:
+    tuple val(meta), path('*.recal.bam')            , emit: bam
+    tuple val(meta), path('*.recal.bai')            , emit: bai
+    tuple val(meta), path('*recal_data.table')      , emit: recal_pre
+    tuple val(meta), path('*recal_data.table.post') , emit: recal_post
+    tuple val(meta), path('*recal_result.csv')      , emit: recal_csv
+    path  "versions.yml"                            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def input  = bam ? '-i ' + bam.sort().join(' -i ') : ''
+    def ref    = fasta ? "-r $fasta" : ''
```

Comment:
QualCal requires the ref file. So it can directly be given in the script section.  If no fasta is given, Sentieon raises an error itself (complaining missing reference). So I think this function is not needed here (and probably in other sentieon driver modules). 
Or keeping this as it is,  at least if the check returns a false value, it should raise some warning or error message!